sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -1474,14 +1454,7 @@ def test_dataframe_agg_method(self, agg_method):
     df = pd.DataFrame({'A': [1, 2, 3, 4], 'B': [2, 3, 5, 7]})
 
     nonparallel = agg_method in (
-        'quantile',
-        'mean',
-        'describe',
-        'median',
-        'sem',
-        'mad',
-        'kurtosis',
-        'kurt')
+        'quantile', 'mean', 'describe', 'median', 'sem', 'mad', 'kurt')
```

Comment:
Can you make it so we use your parallelizable implementation for `kurt` as well?